src/Keboola/StorageApi/Client.php

```diff
@@ -2439,9 +2463,13 @@ public function apiDeleteParams($url, $data)
      */
     public function apiDeleteParamsJson(string $url, array $data = [])
     {
-        return $this->request('delete', $url, [
-            'json' => $data,
-        ]);
+        $options = [];
+        if (empty($data)) {
+            $options['json'] = new stdClass();
+        } else {
+            $options['json'] = $data;
+        }
```

Comment:
Ještě se mi moc nezdá, že jsi to tady nakopíroval 3x...

Reply:
presunuto do `request`